buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchJavaPlugin.java

```diff
@@ -280,14 +280,6 @@ public static void configureCompile(Project project) {
                 compilerArgs.add(targetCompatibilityVersion.getMajorVersion());
             });
         });
-
-        project.getPluginManager().withPlugin("com.github.johnrengelman.shadow", plugin -> {
```

Comment:
This is not required anymore as we 1) now compile by default against the plain compiled classes folders (since we use java-library plugin now) and 2) the runtime config should use the proper shadowjar

Reply:
This _might_ have implications on the generated POM file as well. I'd confirm that it looks the same before/after this change.